raiden/assetmanager.py

```diff
@@ -119,10 +119,12 @@ def register_channel(self, netting_channel, reveal_timeout):
         our_state = ChannelEndState(
             channel_details['our_address'],
             channel_details['our_balance'],
+            self.raiden.chain.block_number,
```

Comment:
This was already done when you reviewed this, no? At least it's done in my latest commit from before you made the comment.